content/docs/platform/inbox/react/components/inbox.mdx

```diff
@@ -299,3 +299,25 @@ function Novu() {
   );
 }
 ```
+
+### Data object
```

Comment:
Please add a page about typesafety of data and the Notification Data interface.

Reply:
Added it below the current section.